### PR TITLE
WIP: add push mqtt5 props card

### PR DESCRIPTION
**Pull request by oceanlvr**
### PR Checklist

If you have any questions, you can refer to the [Contributing Guide](https://github.com/emqx/MQTTX/blob/master/.github/CONTRIBUTING.md)

#### What is the current behavior?

Please describe the current behavior and link to a relevant issue.

#### Issue Number

Example: \#123

#### What is the new behavior?

Please describe the new behavior or provide screenshots.

#### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

#### Specific Instructions

Are there any specific instructions or things that should be known prior to review?

### CODE REVIEW

**Comment by oceanlvr** [comment]
image
@ysfscream

**Comment by ysfscream** [comment]
@oceanlvr Good Job! But sorry, we'll use the first style.

image

**Comment by oceanlvr** [comment]
image

So sorry for my delay commit. @ysfscream

**Comment by oceanlvr** [comment]
Block by MQTT.js publish option.

* https://github.com/oceanlvr/mqtt5-publish-props
* https://github.com/mqttjs/MQTT.js/issues/1357

**Comment by ysfscream** [comment]
@oceanlvr What properties are not received? I test user properties is normal, I see your issue using MQTT broker, not EMQX, is your MQTT broker not support v5.0?

File: src/components/MsgPublish.vue [hunk 1]
```diff
@@ -1,5 +1,88 @@
 <template>
   <div class="msg-publish message">
+    <div class="publish-top">
+      <div v-if="showDialog">
+        <el-card class="box-card">
+          <el-form ref="form" label-width="175px" label-position="left" :model="MQTT5Props" :rules="rules">
+            <el-row class="form-row" :gutter="20">
+              <el-col :span="24">
+                <el-form-item :label="`Content Type`" prop="contentType">
```
Comment (ysfscream) [inline src/components/MsgPublish.vue:9]: Why don't use a normal string? v-bind is unnecessary.

```
label="Content Type"
```

Comment (ysfscream) [reply src/components/MsgPublish.vue:9]: All of that in this form.

File: src/components/MsgPublish.vue [hunk 2]
```diff
@@ -1,5 +1,88 @@
...
+                      <el-button icon="el-icon-delete" class="btn-delete" type="text" @click="deleteItem(index)" />
+                    </div>
+                  </div>
+                </el-form-item>
+              </el-col>
+            </el-row>
+          </el-form>
+          <div class="meta-btn">
+            <el-button size="mini" type="outline" @click="submitForm">save</el-button>
+            <el-button size="mini" type="outline" @click="resetForm">reset</el-button>
```
Comment (ysfscream) [inline src/components/MsgPublish.vue:81]: Translate `save` and `reset`.

**Review by ysfscream — APPROVED** [review]

**Pull request merged**